### PR TITLE
UI/UX improvements

### PR DESCRIPTION
**Pull request by chamoda**
* Show more details after register.
* Ability to unregister.
* Allow users to change next_print without printer capability.
* Loading correct printer list by dynamically loading printer type when machine type change.

### CODE REVIEW

**Comment by markwal** [comment]
@vmorrisPolar Some of this overlaps with work you're doing, right?

**Comment by vmorrisPolar** [comment]
@markwal Chamoda is the one helping us out, I am not technical, he's been doing the work for us. I just tested it out locally and it offers a much better registration and unregistration experience.

File: octoprint_polarcloud/templates/polarcloud_settings.jinja2 [hunk 1]
```diff
@@ -109,4 +130,32 @@
...
+                    {{ _('Failed to unregister OctoPrint with the Polar Cloud.') }}
+                    <span data-bind="html: unregistrationFailedReason"></span>
+                </div>
+            </div>
+            <div>
+                <p>
+                    {{ _('Are you sure you want to unregister this printer? This will mark your serial number as "Deleted" in the cloud and allow someone else to register this octoprint instance with their own username and PIN.') }}
+                </p>
+                <p>
+                    {{ _('Please not that if you try to register it again using your same credentials, you will be granted the same serial number you had before.') }}
```
Comment (markwal) [inline octoprint_polarcloud/templates/polarcloud_settings.jinja2]: spell check: not->note

Comment (markwal) [reply octoprint_polarcloud/templates/polarcloud_settings.jinja2]: @chamoda, @vmorrisPolar

**Review by markwal — APPROVED** [review]
Looks good to me.

**Pull request merged**